backend/lib/mes-aides/emails/email-render.ts

```diff
@@ -133,6 +142,12 @@ export default async function emailRender(emailType, followup) {
         text: values[0],
         html: values[1].html,
       }
+    } else if (emailType === EmailType.tousABordNotification) {
+      return {
+        subject: `Déplacez-vous pour 5€ / mois sur votre réseau bus et TER`,
```

Comment:
J'ai l'impression qu'il manque les espaces insécables ici
```suggestion
        subject: `Déplacez-vous pour 5€ / mois sur votre réseau bus et TER`,
```

Reply:
Good, je les ai ajoutés